grc/core/params/dtypes.py

```diff
@@ -39,15 +46,15 @@ class ValidateError(Exception):
 
 
 @validates('id')
-def validate_block_id(param, black_listed_ids):
+def validate_block_id(param, black_listed_ids: List[str]):
```

Comment:
Can we type hint this? If there's many types, we could return `-> Union[possible type, possible type, ...]` and if necessary, make that a type alias?

Reply:
We could, but the value is thrown away anyways, as far as I'm aware; maybe we just remove the `return value` from all these `validate_*` and `-> None`?

https://github.com/gnuradio/gnuradio/blob/dc5d108344856c94fd28ab8c1d0170be2b714678/grc/core/params/param.py#L164-L169

Reply:
Let me actually quickly try this; will have to squash things here anyways

Reply:
Did the annotation: most of these validators weren't returning anything, and for the other two, since they are 100% only used through the `validators` "factory", that return value would never be usable. Hence, removed, and annotated all functions `-> None`.